Feat 6384326779 fellowships application page updates

RyanCrux opened the pull request:
## What type of PR is this? (check all applicable)

- [x] Feature
- [ ] Bug Fix
- [ ] Hot Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

updated all faqs and entire timeline. 

- Monday Task #
https://blackberg-group.monday.com/boards/1931719392/pulses/6384326779

## QA Instructions, Screenshots, Recordings
Look at the ticket linked above. Find attached word and PP documents. check that content matches for both the faqs and timeline on the fellowship-application page on dev-innovation. 

additionally, test all links found within the text. I copied the hyperlinks but the pages were blocked. I'm not sure if it's a permissions thing on my end or if the provided links were incorrect.


### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_
- [ ] Semantic HTML implemented?
- [ ] Keyboard operability supported?
- [ ] Checked with [Google Lighthouse](https://developer.chrome.com/docs/lighthouse/overview) and addressed `Critical` and `Serious` issues?
- [ ] Color contrast tested?


## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized 

## Changed Files in this Pull Request ##
- _site/fellowship-application/index.html
 - fellowship-application/index.html

RyanCrux commented:
Slipping in text edit changes from another ticket under the same fellowships domain. Please also review: https://blackberg-group.monday.com/boards/1931719392/pulses/6384326779 before approving

ecstech-admin reviewed (changes requested):
Need to update "heading, the text paragraph under the heading"

April 23, 25, 30 + May 9: PPT shows "Meet the Fellows" as bold

Content out of order: June 14, May 21 + June 6, May 14

Content out of order: Sept 2024, Aug 2024

Alignment seems to be off on some of the background lines (yellow boxes on attached screen shot.)
Background lines need z-index adjustments (red boxes on attached screen shot. Should be behind text)

Confirmed that links require you to be logged into VA and provide access to "FY2025 Fellowships Applications Sessions" form

![image](https://github.com/blackberggroup/VA-FELLOWSHIPS/assets/119957662/914bb44a-ac67-40cd-8551-8123048498cb)

**FAQ**
Overview and Eligibility / Do I need to have VA employment to apply? / 3rd paragraph: numbers (0160 or 0152) are bold in Word doc
Overview and Eligibility / How can I apply to VHA Innovation Ecosystem Fellowships? / add commas after "2024" to be consistent with rest of FAQ's grammar style for dates.
Overview and Eligibility / Can an application be submitted on behalf of another individual? / rm padding-x-5 class
Overview and Eligibility / correct last 2 bullets in this section (see Word doc)

RyanCrux commented:
Need to update "heading, the text paragraph under the heading"
     - Double checked this, both the heading the text paragraph under the heading are accurate to the new PP

April 23, 25, 30 + May 9: PPT shows "Meet the Fellows" as bold
     -Done

Content out of order: June 14, May 21 + June 6, May 14
Content out of order: Sept 2024, Aug 2024
     - good catch on the date direction of the timeline. I edited them in order on the page but missed that they snake back from the right side. Done 

Alignment seems to be off on some of the background lines (yellow boxes on attached screen shot.)
Background lines need z-index adjustments (red boxes on attached screen shot. Should be behind text)
     - This is going to be screensize and/or browser specific (image attached of what I'm seeing). I'm using ff and can't find the exact breakpoint. I found a few things that could be tightened up but this whole thing is a little wonky. I think this is beyond scope for this ticket. I'll bring it up to Will to see if it's something he wants me to spend time retooling or editing after we get these changes live. 

Confirmed that links require you to be logged into VA and provide access to "FY2025 Fellowships Applications Sessions" form
     - cool, we'll see if they say anything about this when they review.

Edits are pushed, 
Thanks JIm
<img width="834" alt="Screenshot 2024-04-03 at 5 09 31 PM" src="https://github.com/blackberggroup/VA-FELLOWSHIPS/assets/153299181/78f02cfb-34d5-4bd4-9d21-6f293ffcad09">

RyanCrux commented:
FAQ
Overview and Eligibility / Do I need to have VA employment to apply? / 3rd paragraph: numbers (0160 or 0152) are bold in Word doc
     -Done

Overview and Eligibility / How can I apply to VHA Innovation Ecosystem Fellowships? / add commas after "2024" to be consistent with rest of FAQ's grammar style for dates.
     - Done ( I feel a little uneasy about editing provided text )
     
Overview and Eligibility / Can an application be submitted on behalf of another individual? / rm padding-x-5 class
     - Done

Overview and Eligibility / correct last 2 bullets in this section (see Word doc)
     -Done

will-bbg commented on fellowship-application/index.html:
```diff
@@ -99,73 +99,89 @@ <h1>VHA Innovation Ecosystem Fellowship Applications</h1>
-                                        <h2 class="card__header h4">Phase 2</h2>
-                                        <p class="card__body">VHA IE hosts an information call for prospective applicants.</p>
+                                        <h2 class="card__header h4">April 23, 25, 30 + May 9</h2>
+                                        <p class="card__body"><span style="font-weight: bold;">Meet the Fellows</span> Information Session. <a href="https://forms.office.com/Pages/ResponsePage.aspx?id=Ixtf6a-r7kWCHberJRqzv8Qc7YLkpLNJptNkv8H4TKNUOVhXM0o5RUE5UFJNNTk5VU1WQVVJNFVCMi4u">Sign up here</a>.
```
The use of "inline styles" is not prohibited. The USWDS has text style classes if you want to use a `<span>` tag. If not, the `<strong>` will suffice.

will-bbg replied:
https://designsystem.digital.gov/utilities/text-styles/

will-bbg reviewed (changes requested):
Please make links the same color as link in the CTA section below. 

<img width="807" alt="Screenshot 2024-04-04 at 9 24 31 AM" src="https://github.com/blackberggroup/VA-FELLOWSHIPS/assets/145696026/bd2db1c9-3ddb-4864-8f5e-bd853b14a719">

As for the lines connecting the cards, yes, make adjustments so text is not cut off.

ecstech-admin commented:
Regarding Profile edits...easier to provide than describe. JSON edits for MTran

job: "Marriage and Family Therapist",
    job2: "Reproductive Health Program Manager",
    job_description: "",
    office_title: "National Lead, PREPARe",
    office_location: "Orlando VA Healthcare System",
    office_title2: "",
    office_location2: "",
    project: "Project from Thrive to Perinatal Reproductive Education Planning and Resources (PREPARe)",

ecstech-admin reviewed (changes requested):
**Application Timeline**
Looks good: lines no longer interfering with text. Some lines still don't vertically align center.

**Fellows JSON - Add back MTran bio**
bio: "Melissa Tran is a Marriage and Family Therapist and the Military Sexual Trauma Coordinator at Orlando VA Healthcare System. She leads the Perinatal Reproductive Education Planning and Resources (PREPARe) service, which has garnered national recognition through the achievement of the 2021 Dr. Robert L Jesse Award for Innovation. Ms. Tran is also an instructor for Nova Southeastern Kiran C Patel and University of Central Florida Colleges of Medicine and supports the diversity, equity, and inclusion team at OVAHCS.",
    bio2: "Perinatal Reproductive Education Planning and Resources (PREPARe) is a multidisciplinary approach to meet the needs of our perinatal Veterans. The innovation was developed due to a lack of comprehensive and easily accessible perinatal services available to Veterans. Services are scattered between VA, Community Care or not available at all. Veterans must pay out of pocket and request reimbursement for such services as lactation counseling covered under the Maternity Care benefit. PREPARe is a one stop shop wraparound approach where services typically offered in the community are now offered by trained VA specialists. These services include whole health, mental health, nutrition, physical therapy, women's health, chaplain care, and more."

**Fellows JSON - rm extra, erroneous alt in MTran**
alt: "Jacquelyn Paykel, M.D., M.B.A."

RyanCrux commented:
Updated Jim's additional edits.

Also merged this ticket in with this PR: https://blackberg-group.monday.com/boards/1931719392/pulses/6411718157

uncommitted guide block, Added a new pdf and booklet thumbnail image.

@ecstech-admin please review and I'll get this pushed and back to Will. Thanks!

ecstech-admin reviewed (approved):
Put application guide PDF in /assets/documents.

After that, good to merge.

RyanCrux commented:
Created new directory, adjusted pdf location path. Please review

ecstech-admin reviewed (approved):
Approved
